app/models/FileInfo.scala

```diff
@@ -18,7 +18,7 @@ package models
 
 import play.api.libs.json._
 
-case class FileInfo(name: String, evidenceType: Option[EvidenceType])
+case class FileInfo(name: Option[String], evidenceType: Option[EvidenceType])
```

Comment:
Is it necessary to make `name` an Option. I'm not sure that name should/could ever be None.

Reply:
it needs to be optional. We can have a huddle to discuss why if you'd like...in short, we now need to save the evidence type before the file is uploaded.